servo/connectors/kubernetes.py

```diff
@@ -2550,7 +3140,7 @@ async def create(
         # NOTE: Currently only supporting one container
         assert len(deployment_config.containers) == 1, "CanaryOptimization currently only supports a single container"
         container_config = deployment_config.containers[0]
-        main_container = deployment.find_container(container_config.name)
+        main_container = await deployment.get_tuning_container(container_config)
```

Comment:
`main_container` is assigned to the return value of a method call `get_tuning_container`? Can't be right